Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_046.SqlConnectionString.sarif

```diff
@@ -421,11 +421,53 @@
                   "uriBaseId": "SRC_ROOT"
                 },
                 "region": {
-                  "startLine": 45,
+                  "startLine": 37,
+                  "startColumn": 38,
+                  "endLine": 37,
+                  "endColumn": 129,
+                  "charOffset": 2280,
+                  "charLength": 91,
+                  "snippet": {
+                    "text": "Data Source=servername17;Initial Catalog=database17;User ID=username17;Password=password17\""
```

Comment:
>\" [](start = 119, length = 2)

since we are using order insensitive, its hard to create the refine, since each part contains the "terminator".